cLIMS/organization/export.py

```diff
@@ -1256,4 +1255,4 @@ def exportDCIC(request):
             return response
     else:
         messages.error(request, 'Please add the all experiments into biological/technical replicate experiment set!')
-        return HttpResponseRedirect('/detailProject/'+request.session['projectId'])
+        return HttpResponseRedirect('/detailProject/'+request.session['projectId'])
```

Comment:
there is a newline symbol that got lost during editing
not a big deal or problem at all, but ...
when someone else if reviewing a PR, i.e. some code changes - it is easy to get lost when there a ton of such small changes
We should discuss git-best practices at some point.

Reply:
Makes sense.